src/main/target/BLUEJAYF4/target.c

```diff
@@ -31,4 +34,15 @@ const timerHardware_t timerHardware[USABLE_TIMER_CHANNEL_COUNT] = {
     { TIM3, IO_TAG(PB1), TIM_Channel_4, TIM_USE_MOTOR | TIM_USE_LED, 1, GPIO_AF_TIM3, DMA1_Stream2, DMA_Channel_5, DMA1_ST2_HANDLER }, // S5_OUT
     { TIM3, IO_TAG(PB0), TIM_Channel_3, TIM_USE_MOTOR,               1, GPIO_AF_TIM3, DMA1_Stream7, DMA_Channel_5, DMA1_ST7_HANDLER }, // S6_OUT
 };
+*/
+
+const timerHardware_t timerHardware[USABLE_TIMER_CHANNEL_COUNT] = { 
+    DEF_TIM(TIM8, CH2, PC7, TIM_USE_PPM,                 0, 0 ), // PPM IN
```

Comment:
When I posted the comment I hadn't thought enough. This is a suffix to a #define, so, as you point out, it can be done, but it's a pain and also error prone.

So I wouldn't do this - disregard my original comment.